infra/blueprint-test/pkg/kpt/kpt.go

```diff
@@ -67,3 +73,18 @@ func (k *CmdCfg) RunCmd(args ...string) string {
 	}
 	return op
 }
+
+// findKptfile discovers Kptfile of the root package from slice of nodes
```

Comment:
Can we upstream this into kptutil?

Reply:
Yes, I have that as an AI in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/issues/1087. We found a spot but there is some heavy refactoring being done on the codebase. @yuwenma suggested we can upstream once that PR was been merged.